local/local.go

```diff
@@ -683,3 +685,13 @@ func v1Config(inspect types.ImageInspect) (v1.ConfigFile, error) {
 		Config: config,
 	}, nil
 }
+
+// ensureReaderClosed drains and closes and reader, returning the first error
+func ensureReaderClosed(r io.ReadCloser) error {
+	var err error
+	_, err = io.Copy(ioutil.Discard, r)
```

Comment:
`_, err :=`

Reply:
(unless the separated declaration was stylistic and not the result of refactoring)

Reply:
fixed